src/hooks/usePriceRoundEvents.ts

```diff
@@ -8,7 +8,10 @@ import {
 } from "web3/queryVotingContractEvents";
 
 const signer = provider.getSigner();
-const contract = createVotingContractInstance(signer);
+const contract = createVotingContractInstance(
+  signer,
+  process.env.REACT_APP_TESTING_GANACHE ? "1337" : "1"
```

Comment:
how can this also handle kovan

Reply:
Good catch. I didn't swap this over to using determineBlockchainNetwork().